CMakeLists.txt

```diff
@@ -751,15 +751,15 @@ set(PACKAGE_URL "https://github.com/opencollab/arpack-ng/")
 set(ARPACK_PC_LIBS_PRIVATE)
 foreach(lib ${LAPACK_LIBRARIES})
     get_filename_component(libname ${lib} NAME)
-    string(REGEX REPLACE "^lib([^.]+).*$" "-l\\1" libname ${libname})
+    string(REGEX REPLACE "^(lib)?([^.]+).*$" "-l\\2" libname ${libname})
     list(APPEND ARPACK_PC_LIBS_PRIVATE "${libname}")
 endforeach()
 string(REPLACE ";" " " ARPACK_PC_LIBS_PRIVATE "${ARPACK_PC_LIBS_PRIVATE}")
 
 set(PARPACK_PC_LIBS_PRIVATE)
```

Comment:
PARPACK_PC_LIBS_PRIVATE looks like to be unused in CMakeLists.txt !?... Do you use it? If no, kill it

Reply:
Same as above.
It is used in `PARPACK/SRC/MPI/parpack.pc.in`.

Reply:
OK, not used in CMakeLists.txt but used elsewhere!